dot/sync/tip_syncer_integration_test.go

```diff
@@ -26,9 +26,10 @@ func newTestTipSyncer(t *testing.T) *tipSyncer {
 		trie.EmptyHash, trie.EmptyHash, 200, types.NewDigest())
 	require.NoError(t, err)
 
-	bs := new(syncmocks.BlockState)
-	bs.On("GetHighestFinalisedHeader").Return(finHeader, nil)
-	bs.On("HasHeader", mock.AnythingOfType("common.Hash")).Return(true, nil)
+	ctrl := gomock.NewController(t)
+	bs := NewMockBlockState(ctrl)
+	bs.EXPECT().GetHighestFinalisedHeader().Return(finHeader, nil).AnyTimes()
+	bs.EXPECT().HasHeader(gomock.AssignableToTypeOf(common.Hash{})).Return(true, nil).AnyTimes()
```

Comment:
1. Try to find the expected hash value for the argument (just run it against a dummy `common.Hash{1}` and it will log the one to expect)
2. Try to find the number of times it's called.